client/mysqladmin.cc

```diff
@@ -1428,10 +1428,12 @@ static void usage(void)
   version		Get version info from server");
 }
 
+#define SHOW_MSG_LEN (FN_REFLEN + 20)
+#define UINPUT_BUF_LEN 10
```

Comment:
Why introduce this \#define UINPUT_BUF_LEN? This is unrelated to sprintf/snprintf(), should be done as a separate patch (or why do it at all, the code is simpler without the \#define, it's only used in one place)